.github/labels.yml

```diff
@@ -229,25 +218,13 @@
 - name: hint/needs-community-input
   color: 0623cc
   description: "Hint: Needs Community Input"
```

Comment:
When would we use this one?

Reply:
Whenever we're making a change that impacts users and want their feedback. Not only maintainers.

Reply:
I can see only one issue labeled as such so far - https://github.com/testground/testground/issues/691

It's a bit unclear to me what are the actions we're waiting for if something is labelled with this (do we go off GitHub to find testground users? do we mention someone?) and when we can remove this label (how many voices from the community is enough?).

I think the discussion I'm turning this into is out of scope for this PR (thanks for doing this btw, I think it's a great initiative).

So I'm voting for resolving/closing this thread here.

Reply:
Agreed, we dropped something like 25 labels, if we have one left, that's already a great improvement,
thanks for the feedback!

For what it's worth if/when we discuss this again: when it comes to community involvement & working in the open, I think we should rely on the experience from other projects, with more activity. We can find examples of this label in [go-libp2p](https://github.com/libp2p/go-libp2p/issues?q=is%3Aopen+is%3Aissue+label%3Aneed%2Fcommunity-input) and [kubo](https://github.com/ipfs/kubo/issues?q=is%3Aopen+is%3Aissue+label%3Aneed%2Fcommunity-input).